src/server/req_modify.c

```diff
@@ -600,18 +600,10 @@ modify_job_attr(job *pjob, svrattrl *plist, int perm, int *bad)
 	/* OK, if resources changed, reset entity sums */
 
 	if (changed_resc) {
-		(void)set_entity_resc_sum_max(pjob, NULL,
-			&newattr[(int)JOB_ATR_resource],
-			INCR);
-		(void)set_entity_resc_sum_queued(pjob, NULL,
-			&newattr[(int)JOB_ATR_resource],
-			INCR);
-		(void)set_entity_resc_sum_max(pjob, pjob->ji_qhdr,
-			&newattr[(int)JOB_ATR_resource],
-			INCR);
-		(void)set_entity_resc_sum_queued(pjob, pjob->ji_qhdr,
-			&newattr[(int)JOB_ATR_resource],
-			INCR);
+		account_entity_limit_usages(pjob, NULL,
+				&newattr[(int)JOB_ATR_resource], INCR, ETLIM_ACC_ALL_RES);
+		account_entity_limit_usages(pjob, pjob->ji_qhdr,
+				&newattr[(int)JOB_ATR_resource], INCR, ETLIM_ACC_ALL_RES);
```

Comment:
Here we have to `INCR` only res limit attributes. i.e invoke only `set_entity_resc_sum_*` in `account_entity_limit_usages()`. Hence passing `ETLIM_ACC_ALL_RES` as flag.

Reply:
if qalter lowered the value of the resource then would that not change the operator to DECR?

Reply:
I'm not sure, but by looking at the logic of `set_single_entity_res()` it seems to handle the DECR. But anyways I will keep this scenario in mind while expanding the PTL test suite.

Reply:
@Shrini-h did you cover this scenario in a test?

Reply:
Fixed in rc12 https://github.com/PBSPro/pbspro/pull/807/commits/70bdf5747dfa88cd44b20ac99ef93781c4a3d5a0